src/Microsoft.Extensions.Caching.SqlServer/SqlServerCacheOptions.cs

```diff
@@ -37,6 +46,11 @@ public class SqlServerCacheOptions : IOptions<SqlServerCacheOptions>
         /// </summary>
         public string TableName { get; set; }
 
+        /// <summary>
+        /// The default options for a new cache entry. It has SlidingExpiration set for 20 minutes by default.
+        /// </summary>
+        public DistributedCacheEntryOptions DefaultCacheEntryOptions { get; set; }
```

Comment:
Don't use this type here, it's not future proof, just add `DefaultSlidingExpiration`.